Sources/Mantis/Helpers/TransformRecord.swift

```diff
@@ -0,0 +1,85 @@
+import UIKit
+
+enum TransformType {
+    case resetTransforms
+    case transform
+}
+
+class TransformRecord: NSObject {
```

Comment:
I can still see public properties and functions in this class, please check if they are necessary.

Reply:
changes made.